src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -116,6 +126,7 @@ const ColonyHome = ({
   const [filteredDomainId, setFilteredDomainId] = useState(0);
   const [isTaskBeingCreated, setIsTaskBeingCreated] = useState(false);
   const [showRecoverOption, setRecoverOption] = useState(false);
+  const [activeTab, storeActiveTab] = useState('tasks');
```

Comment:
Let's follow the same convention here - `setActiveTab` instead of `storeActiveTab`

Also, could we make this typing more strict? Something like `... = useState<'tasks' | 'transactions'>('tasks');`